benchmark/bench_etree.py

```diff
@@ -7,7 +7,7 @@
                        serialized, children, nochange)
 
 TEXT  = "some ASCII text"
-UTEXT = u"some klingon: \F8D2"
+UTEXT = u"some klingon: \uF8D2"
```

Comment:
this is _technically_ a behaviour change -- though I think the original code was a mistake given the `u` prefix